sql_generators/glean_usage/templates/baseline_clients_first_seen_v1.query.sql

```diff
@@ -71,7 +71,7 @@ _previous AS (
   FROM
     `{{ first_seen_table }}`
   WHERE
-    first_seen_date > "2010-01-01"
+    first_seen_date < @submission_date
```

Comment:
Not sure if there is a reason to keep the 2010-01-01, maybe in case we need to do a full backfill?

```suggestion
    first_seen_date > "2010-01-01" AND
    first_seen_date < @submission_date
```

Reply:
- Should this also be done for the other branch of the `if`/`else`? (e.g. line 49)
- Suggest putting `AND` at the beginning of the subsequent line.

Reply:
Yeah, good point. I'll add it there as well